checks/http/microsoft.rb

```diff
@@ -1536,6 +1538,8 @@ def owa_to_exchange_version(owa_version)
             out = { version: '2019', update: 'Cumulative Update 8' }
           elsif ['15.2.858', '15.2.858.5'].include?(owa_version)
             out = { version: '2019', update: 'Cumulative Update 9' }
+          elsif ['15.2.922', '15.2.922.7'].include?(owa_version)
+            out = { version: '2019', update: 'Cumulative Update 10' }
```

Comment:
I probably missing do we want the 15.2.922.13?
![image](https://user-images.githubusercontent.com/7880605/132985164-e4368ac3-330f-472f-ad49-b3aa63846460.png)